cypress/e2e/cloud/operator.test.ts

```diff
@@ -49,10 +49,13 @@ describe('Operator Page', () => {
       .invoke('attr', 'placeholder')
       .should('contain', 'Filter accounts by email')
 
-    cy.getByTestID('operator-resource--searchbar').type('asalem', {
-      force: true,
-      delay: 50,
-    })
+    cy.focused()
```

Comment:
i don't think this does anything by itself if you don't use the yielded element. i think `cy.focused` by itself is like doing `array.map(operation)` and not storing the return value in an array

https://docs.cypress.io/api/commands/focused#Usage